docs/en/ingest-management/release-notes/release-notes-8.16.asciidoc

```diff
@@ -0,0 +1,237 @@
+// Use these for links to issue and pulls.
+:kibana-issue: https://github.com/elastic/kibana/issues/
+:kibana-pull: https://github.com/elastic/kibana/pull/
+:beats-issue: https://github.com/elastic/beats/issues/
+:beats-pull: https://github.com/elastic/beats/pull/
+:agent-libs-pull: https://github.com/elastic/elastic-agent-libs/pull/
+:agent-issue: https://github.com/elastic/elastic-agent/issues/
+:agent-pull: https://github.com/elastic/elastic-agent/pull/
+:fleet-server-issue: https://github.com/elastic/fleet-server/issues/
+:fleet-server-pull: https://github.com/elastic/fleet-server/pull/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-8.16.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+// begin 8.16.0 relnotes
+
+[[release-notes-8.16.0]]
+== {fleet} and {agent} 8.16.0
+
+Review important information about the {fleet} and {agent} 8.16.0 release.
+
+[discrete]
+[[security-updates-8.16.0]]
+=== Security updates
+
+{fleet-server}::
+* Update {fleet-server} Go version to 1.23.1. {fleet-server-pull}3924[#3924]
+
+[discrete]
+[[new-features-8.16.0]]
+=== New features
+
+The 8.16.0 release Added the following new and notable features.
+
+{fleet}::
+* Support content packages in UI. {kibana-pull}195831[#195831]
+* Advanced agent monitoring options UI for HTTP endpoint and diagnostics. {kibana-pull}193361[#193361]
+* Add option to have Kafka dynamic topics in outputs. {kibana-pull}192720[#192720]
+* Add support for GeoIP processor databases in Ingest Pipelines. {kibana-pull}190830[#190830]
+* Add agentless UX creation flow. {kibana-pull}189932[#189932]
```

Comment:
@kpollich do we want anything to do with agentless in the release notes yet, considering it isn't usable or really announced yet?